### ISSUE TITLE
pull-request pass credentials from cmd

### ISSUE THREAD

**Issue by bmihaescu** [seveas/git-spindle#179]
Hi,

Is there a way to pass the username and password for the github repo from the cmd ?

Something like this
git-hub pull-request --issue="merge into master" staging:master -user USERNAME -pass PASSWORD

Or to get them from a file

Thanks

**Comment by seveas** [comment]
git-hub would rather not know your username and password at all :smile: It asks for them only once: to create a token for further authentication, this token is then stored either in a file, or in your configured credential helper.

Passing a username and password on the commandline is insecure in shared environments: everybody can read those while the process is running. Reading the username and password from a file would mean that they need to be in a file in the first place, which I'd rather avoid too. The token already ends up in a file.
